MyPandoc.cabal

```diff
@@ -26,6 +26,9 @@ source-repository head
 library
   exposed-modules:
```

Comment:
This should changed in package.yml, the .cabal file is auto-generated.

Reply:
done